xbmc/platform/darwin/tvos/input/LibInputTouch.mm

```diff
@@ -257,6 +276,34 @@ - (void)SiriSelectHandler:(UITapGestureRecognizer*)sender
   }
 }
 
+- (void)pageUpPressed:(UITapGestureRecognizer*)sender
+{
+  switch (sender.state)
+  {
+    case UIGestureRecognizerStateEnded:
```

Comment:
what if user would hold the button? Does it make any sense from usability POV?

Reply:
I did not try to implement the "Long press" and "Double tap" versions of page up and page down. So ATM nothing happen.
Should I? I don't know if a long press on channel up is useful...

Reply:
np, can always be added later. Having something as a start is already nice.